README.md

```diff
@@ -1,53 +1,78 @@
+* User authorizes applications to read/write/control data in pod
 
-After installing the dependencies from package.json, run the following to start a local live server to view the application:
+#### To become familiar with how pods work:
+* Go to [Kevin's Solid Experiment](https://github.com/codeforpdx/PASS/tree/km/solid/experiments/experiments/solid) on GitHub.
+* Clone repository and follow the [instructions](https://github.com/codeforpdx/PASS/tree/km/solid/experiments/experiments/solid/01-one-server).
+* Feel free to play around with the pod and check out the second use case [here](https://github.com/codeforpdx/PASS/tree/km/solid/experiments/experiments/solid/02-different-file-browser-application).
 
-```shell
-npx parcel ./src/index.html
-```
+### Inrupt JavaSctipt Client Libraries
 
-The local live server would be located in port 1234:
+Inrupt is currently leading the way for development with Solid. Check out their documentation [here](https://docs.inrupt.com/developer-tools/javascript/client-libraries/)
 
-```shell
-http://localhost:1234/
-```
+### RDF (Resource Description Framework) and Turtle (ttl files)
 
-Latest version of this build require users to login to their Pod through [https://opencommons.net](https://opencommons.net)
+The Resource Description Framework (RDF) is a framework for expressing information about resources. Resources can be anything, including documents, people, physical objects, and abstract concepts. RDF is intended for situations in which information on the Web needs to be processed by applications, rather than being only displayed to people. Read more about RDF [here](https://www.w3.org/TR/rdf11-primer/). See how RDF is used within Solid [here](https://solidproject.org/developers/vocabularies/well-known/core).
+
+There is a textual syntax for RDF called Turtle that allows an RDF graph to be completely written in a compact and natural text form, with abbreviations for common usage patterns and datatypes. Read more on the Turtle spec [here](https://www.w3.org/TR/turtle/)
+
+With PASS we are looking into using RDF to link between pods and store metadata in turtle files. We are exploring the use of ttl files to enable organizations to query into an individuals pod to see if certain documents are present, while restricting access to the documents themselves.
+
+
+### The Team
+
+|Back-End Dev Team    | Front-End Dev Team    | UX/UI Team | Project Management Team |
+| ------------------- | --------------------- | -----------| ------------------------|
+| Team Lead - Kevin M.| Team Lead - Jared K.  | Gabby      | Flo                     |
+|                     |                       | Kyle       | Danica                  |
+|                     |                       |            | Wilfred                 |
+|                     |                       |            | Katherine               |
+|                     |      
+
+
+
+## Check out the documentation:
```

Comment:
Could we change this to something like **Additional Resources** or something like **Additional Reading**

Reply:
If you want to make that swap you can after the merge. its up to you. I went  ahead and marked everything as good to go 👍